crates/amaru-kernel/src/lib.rs

```diff
@@ -1189,6 +1200,21 @@ pub fn sum_ex_units(left: ExUnits, right: ExUnits) -> ExUnits {
     }
 }
 
+// FIXME: cleanup clones?
+pub fn to_redeemer_keys(redeemers: &Redeemers) -> Vec<RedeemersKey> {
```

Comment:
Worth implementing `From` ? Same for `script_purpose_to_string `

Reply:
I can't, these are types defined in Pallas so compiler yells at me.

Reply:
We can submit PRs to pallas; can you drop a TODO? this will create great fodder for Sundae Labs to contribute during our mercenary era